src/components/typography/typography.styles.ts

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+
+/**
```

Comment:
Can you cross check the content of your comments here and other places, based on guidance here - [How to write better code comments](https://dev.to/adammc331/todo-write-a-better-comment-4c8c)?